### ISSUE TITLE
MIDPROJECT REVIEW

### ISSUE THREAD

**Issue by blendaf** [juliuscc/speak.exchange#33]
What has to be done before Friday

- [x] Deploy the app
- [x] **Write a nice readme detailing:** 
  - [x] Short description of the project
  - [x] What you have done
  - [x] What you still plan to do
  - [x] Your project file structure 
  - [x] Short description/purpose of each file
  - [x] (Nice to have) A screenshot of the page
- [x] A link to the deployed/running app
- [x] A link to your git repository

What has to be done after Friday (before Tuesday)

- [x] Peer review which includes

  - Review the running app:
    - Focus on layout and interaction, apply HCI evaluation methodology (e.g. Nielsen's heuristics, Shneiderman's golden rules, etc.) that you learned in a basic Human-Computer Interaction course
  - Use the grading criteria matrix and try to find points where the project might not reach a good grade.
  - Reviewing the code on GitHub (you don't need to download or run it):
    - Is the architecture following the principles you learned through the course and/or the guidlines of the framework the group is using?
    - Is the code DRY (Don’t Repeat Yourself)? If there is code duplication (similar segments of code repeated in several places), they should have been extracted in a loop, method or module that can be reused.
    - Naming convention: Are the files, methods and variables named in meaningful way (reflecting their purpose)? Is the same pattern used consistently throughout the code?
    - Code formatting: is the code properly indented and formatted for good
  - Write your peer review in English.
  - Be respectful and polite towards the group.
  - Provide suggestions for improvements, instead of just pointing out flaws.

**Comment by Falcasantos** [comment]
**Short description of the project**
_speak.exchange_ is an online bilingual dictionary, which offers the sub-app _speak.repeat_, which is a spaced-repetition system (SRS) app geared towards learning new words in a foreign language through flashcards. 

**What we have done**
We have so far implemented most of the MVP of _speak.exchange_ in that it currently fetches translations in English to French from the WordReference API, presenting them to the user. These searches can be shared thanks to persistent URLs.

**What we still plan to do**
We will implement a search history and saved wordlist based on user profiling. We are also yet to implement user authentication, user profiles, and the SRS app within _speak.repeat_. We intend to complete most functionality of _speak.exchange_ before the peer review, and to complete _speak.repeat_ post-peer review.

**Comment by blendaf** [comment]
Finished
